certbot/tests/error_handler_test.py

```diff
@@ -26,34 +46,51 @@ def test_context_manager(self):
             with self.handler:
                 raise ValueError
         except ValueError:
-            pass
+            exception_raised = True
+
+        self.assertTrue(exception_raised)
```

Comment:
This is probably a good addition, but what happens if `ValueError` is not raised? This line will blow up because `exception_raised` isn't defined. I'd add a `exception_raised = False` line to the beginning of this method.